webdriver/tests/support/fixtures_bidi.py

```diff
@@ -207,8 +207,8 @@ async def assert_pdf_dimensions(pdf, expected_dimensions):
         png = await render_pdf_to_png_bidi(pdf)
         width, height = png_dimensions(png)
 
-        assert cm_to_px(expected_dimensions["height"]) == height
-        assert cm_to_px(expected_dimensions["width"]) == width
+        assert (height - 1) <= cm_to_px(expected_dimensions["height"]) <= (height + 1)
```

Comment:
Are there specific tests that are failing for you or do all the tests that check for the dimension of the PDF fail? How many pixels do they differ?

Reply:
I just saw the attached image. So it's basically a single pixel too small in all the cases. Other tests like screenshots are using `floor` to calculate the effective height and width. Maybe that's something we should use here as well to be consistent?

Reply:
If I change floor only in `cm_to_px`, some tests still fail. Is there some other place I could have missed, where we should also round down?

Reply:
> If I change it to floor only in cm_to_px, some tests still fail.

They are passing now. I may have mistaken this with some other change. Updated.

Reply:
This is what happens when I use `floor`:

```
>       assert cm_to_px(expected_dimensions["height"]) == height
  │ E       assert 815 == 816
  │ E         +815
  │ E         -816
```

Reply:
I've discussed this topic with @jgraham and we think it will be fine to keep what you have right now. It shouldn't matter too much if there is a different of 2px or 1px by using `floor`.